komodo/release_transpiler.py

```diff
@@ -93,30 +94,38 @@ def main():
   rhel8:
     py36 : # package not included in release
     py38 : 5.11.13+builtin""",
-        formatter_class=RawTextHelpFormatter,
+        formatter_class=argparse.RawTextHelpFormatter,
     )
     matrix_parser.set_defaults(func=combine)
     matrix_parser.add_argument(
-        "--release-base", help="Name of the release to handle", required=True
+        "--release-base",
+        required=True,
+        help="Name of the release to handle (default: None)",
```

Comment:
Why "default: None", but not explicit in `default`? 
combine -> build_matrix_file seems to expect a name for the path.

Same comment for the 3 instances below also.

Reply:
Well spotted! This parser uses `argparse.RawTextHelpFormatter` to handle showing the expected output file format properly. There seems to be no other way to achieve that (newlines are removed), and I couldn't fine a way to combine it with `argparse.ArgumentDefaultsHelpFormatter`, which normally handles the transclusion of the degfaults. So we end up with this -- not great, but produces what we need.